build.gradle

```diff
@@ -0,0 +1,53 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'signing'
+
+repositories {
+  mavenCentral()
+}
+
+task wrapper(type: Wrapper) {
+    gradleVersion = '2.0'
+}
+
+dependencies {
+    compile (
+  'org.threadly:threadly:3.3.0'
```

Comment:
I would change this to be:
'org.threadly:threadly:3.+'

That way it just uses the latest 3.X version, and when we go to 4.X we will just need to be sure we update this project off any deprecated API's